suite/auto-sync/ARCHITECTURE.md

```diff
@@ -1,12 +1,9 @@
-# Auto-Sync
-
-`auto-sync` is the architecture update tool for Capstone.
-Because the architecture modules of Capstone use mostly code from LLVM,
-we need to update this part with every LLVM release. `auto-sync` helps
-with this synchronization between LLVM and Capstone's modules by
-automating most of it.
+<!--
+Copyright © 2022 Rot127 <unisono@quyllur.org>
+SPDX-License-Identifier: BSD-3
+-->
 
-You can find it in `suite/auto-sync`.
+# Architecture of Auto-Sync
```

Comment:
"Architecture of Auto-Sync tool" or "Architecture of Auto-Sync framework". Something like that. Otherwise it sounds *incomplete*.